src/java.naming/share/classes/javax/naming/spi/NamingManager.java

```diff
@@ -123,9 +123,12 @@ static ObjectFactoryBuilder getObjectFactoryBuilder() {
      *    or {@code Referenceable} containing a factory class name,
      *    use the named factory to create the object.
      *    Return {@code refInfo} if the factory cannot be created.
-     *    Under JDK 1.1, if the factory class must be loaded from a location
-     *    specified in the reference, a {@code SecurityManager} must have
-     *    been installed or the factory creation will fail.
+     *    Downloading a factory class from a location specified in the reference
+     *    is not supported out of the box.
```

Comment:
"out of the box" seems a bit informal for the space.
Perhaps...

Downloading a factory class from a location specified in the reference can be supported by a custom implementation of {@link ObjectFactoryBuilder}.

etc.